src/test/scala/edu/colorado/plv/cuanto/example/ScalaSmtlibExample.scala

```diff
@@ -0,0 +1,129 @@
+package edu.colorado.plv.cuanto
+
+package example
+
+import org.scalatest.tagobjects.Slow
+
+import smtlib.{
+  Interpreter
+}
+import smtlib.interpreters.{
```

Comment:
Try to fix the codacity issue for this import (and maybe for the one above).
If we do not figure out what is causing it we can disable the check.